uiowa/uiowa
# Reverse column order for 2, 3, 4 and Page w/sidebar layouts in mobile

Closes #4782 
Closes #2737
# How to test


1. `blt ds --site=admissions.uiowa.edu`
2. `blt frontend`
3. `drush @admissions.local cr`
4. `drush @admissions.local uli`
5. Go to https://admissions.uiowa.ddev.site/event/next-step-iowa-admitted-student-days-0 and minimize window to mobile. Column on R side should display above column on L side. 
6. When you go to add a multicolumn layout (2, 3, 4, page w/sidebar), you should have option to select the reverse mobile display and it should work accordingly 
7. `blt ds --site=sandbox.uiowa.edu`
8. `drush @sandbox.local uli`
9. go to https://sandbox.uiowa.ddev.site/reverse-column-order-mobile and add the 'Reverse column order on mobile' style to the section
10. minimize the window to a mobile breakpoint - make sure highest column number is at top of section
